### DISCUSSION TITLE
Include Vault / Folder information when viewing an item

### DISCUSSION THREAD

**Discussion by slifty** [bitwarden/clients#7388] [Password Manager]
### Code Contribution Proposal

When viewing an item in a Bitwarden I would like to be able to render what vault and folder the item exists in.

This becomes particularly important if I have multiple vaults (or multiple folders folders) and have selected "All Vaults".

**Comment by djsmith85** [comment]
Hey @slifty is this a feature request or code that you would like to contribute to the Bitwarden codebase?

**Reply by slifty** [reply]
@djsmith85 if this is a feature that would be of interest to the maintainers I would be glad to use this as an opportunity to try my hand at implementing it and opening a PR (or PRs)!
